cluster_migration_core/unit_tests/core/test_framework_runner.py

```diff
@@ -97,8 +97,6 @@ def state_file_for_resume(test_workspace):
 TEST_CONFIG_FILE_PATH = "./path/to/test_config.json"
 
 
-# This patch replaces the namespace of the "steps" module with that of this current file
-@mock.patch('upgrade_testing_framework.core.framework_runner.steps', sys.modules[__name__])
```

Comment:
@chelma I wasn't able to determine how this was being used as it didn't seem to affect test cases, lmk if I am missing something here

Reply:
Interesting.  Hypothetically, you just need to swap `upgrade_testing_framework` for `cluster_migration_core`, but I'm confused why it didn't cause test failures.  Can you either perform that swap or delete the line of code?

Reply:
This is removed currently and no issue I can see, just curious if we still actually needed it or not. The framework runner originally had an import for steps with a comment that it was only used for unit tests, which I removed as steps now belongs in UTF so this patch was failing.

Reply:
I guess we didn't need it.  It was something I copied/pasted from my tests in the Vingilot codebase.